fix http Microsoft links (#313)
MicrosoftDocs/iis-docs: iis/web-hosting/configuring-servers-in-the-windows-web-platform/offline-files-for-shared-configuration.md

## More Information

> For Web site content, you may wish to consider using DFS for your file servers. For more information about DFS, go to the Distributed File System documentation at .

Reviewer: ```suggestion
For Web site content, you may wish to consider using DFS for your file servers. For more information about DFS, see [Distributed File System](https://docs.microsoft.com/windows/desktop/dfs/distributed-file-system).
```
